app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     buildToolsVersion '23.0.2'
     defaultConfig {
         applicationId 'fr.neamar.kiss'
-        minSdkVersion 10
+        minSdkVersion 11
```

Comment:
Hmm. How much weight might these additions add to the app? Would it be as
lightweight as it is now? Furthermore, wouldn't these features be slightly
off-topic for a launcher?

My humble opinions, though.

Il 07/ago/2016 19:42, "Alexander Schlarb" <notifications@github.com> ha
scritto:

> In app/build.gradle
> <https://github.com/Neamar/KISS/pull/510#discussion_r73806078>:
>
> > @@ -4,7 +4,7 @@ android {
> >      buildToolsVersion '23.0.2'
> >      defaultConfig {
> >          applicationId 'fr.neamar.kiss'
> > -        minSdkVersion 10
> > +        minSdkVersion 11
>
> Is there any way to keep Android 2.3.3+ compatibility with these changes?
> We discussed this a couple of weeks ago and people came to the conclusion
> that we want to keep SDK 10 compatibility for now. (It's not a problem if
> the documents provider is always disabled for this platform, btw.)
>
> @Neamar <https://github.com/Neamar>: Comments on this?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Neamar/KISS/pull/510/files/11c85a4a67f57428b1fd7961697f2efe47f704bc#r73806078>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APSuVndIGDGDJ0X_WmgIuOM8oR6K4Gywks5qdhj7gaJpZM4JeZI4>
> .
>